### DISCUSSION TITLE
Run curl in bun shell

### DISCUSSION THREAD

**Discussion by ryhkml** [oven-sh/bun#10688] [Q&A] [answered]
I'm confused why Bun shell gives an error response.
```ts
const options = "-sL https://example.com";
const res = await $`curl ${options}`.text();
console.log(res);
```
Same as
```sh
curl -sL https://example.com
```
When executed it returns a:
![Screenshot from 2024-04-30 15-36-06](https://github.com/oven-sh/bun/assets/36887379/be13e080-c35e-442c-bdf1-0c6e4c3cd8e5)
Can anyone explain?

**Comment by truongan07** [comment, marked as answer]
Try this code. Hope to help
```
const options = "-sL https://example.com";
const res = await $`${{raw: `curl ${options}`}}`.text();
console.log(res);
```

**Comment by ryhkml** [comment]
This is very helpful, thanks
